server/svix-server/src/v1/endpoints/attempt.rs

```diff
@@ -238,7 +245,15 @@ async fn list_attempted_messages(
     let into = |(dest, msg): (messagedestination::Model, Option<message::Model>)| {
         let msg =
             msg.ok_or_else(|| err_database!("No associated message with messagedestination"))?;
-        Ok(EndpointMessageOut::from_dest_and_msg(dest, msg))
+        Ok(EndpointMessageOut {
```

Comment:
I kind of preferred being able to do this in one line.

Reply:
Just a comment though. Feel free to merge regardless.

Reply:
No, that's fair. I added that back with a new parameter on `from_dest_and_msg`. Still not quite one line, but less noisy.